### ISSUE TITLE
Home World cannot be properly closed

### ISSUE THREAD

**Issue by JackTheFoxOtter** [Yellow-Dog-Man/Resonite-Issues#380]
### Describe the bug?

This is something I've so far only experienced with my currently favorited home. Attempting to close the world wil disconnect all other connected players, but won't actually close the world. Clicking close world again doesn't work either.

### To Reproduce

Launch Resonite, wait a moment, attempt to close the home world.

### Expected behavior

The home world should close properly.

### Resonite Version Number

2023.10.20.831

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Valve Index

### Log Files

[J4-C - 2023.10.20.831 - 2023-10-21 17_13_22.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13061069/J4-C.-.2023.10.20.831.-.2023-10-21.17_13_22.log)

I pressed F3 before closing the world via the "Close World" facet (discarding changes), but the log doesn't show anything.

### Reporters

@JackTheFoxOtter

**Comment by Nytra** [comment]
Try focusing to a different world such as your local home after you click close world.

**Comment by JackTheFoxOtter** [comment]
> Try focusing to a different world such as your local home after you click close world.

Oh intersting.
Focusing on a different world does indeed cause the world to close after I clicked the "Close World" button.

**Comment by Nytra** [comment]
Yes so I think this has been an issue since Neos. I always had to focus to my local home in order to close my cloud home.

**Comment by MargoSizemore** [comment]
I've experienced this too

**Comment by JackTheFoxOtter** [comment]
Updated title to better reflect the underlying issue.

**Comment by Frooxius** [comment]
How are you closing the home exactly?

**Comment by jae1911** [comment]
I can reproduce that using the shortcut to leave worlds on desktop (CTRL + double tap ESC) which should close the world immediately but doesn't.

**Comment by JackTheFoxOtter** [comment]
Through the world browser. Using the "Close World" facet on Dash's home page does close it immediately, but closing it through the world browser (or the WorldsYou'reIn-Facet) will not close it until you actually focus to a different session.

**Comment by VirgilCore** [comment]
@shiftyscales nope. This bug happens to me with the startup home world too. Its not just the default cloud home, its any wold you have set as "home"

**Comment by VirgilCore** [comment]
When i get home, i will send video and logs. This problem happens for the world opened on engine start.
(Also, no need to be condescending)

**Comment by TisFoolish** [comment]
My boyfriend Bleok also experiences the same thing

[The log I was given](https://github.com/user-attachments/files/22731949/CARLOS.-.2025.9.23.1237.-.2025-10-06.17_11_19.log)

[Video recording](https://cdn.steamusercontent.com/ugc/10964664029696416128/B8901B3B551F75EBF633E78C8122AD9E773D5471/)

Launching the cloud home with Res
